kong/plugins/request-transformer/access.lua

```diff
@@ -142,7 +144,7 @@ local function transform_headers(conf)
 
   if replace then
     for _, name, value in iter(conf.replace.headers) do
-      if headers[name] ~= nil or lower(name) == "host" then
+      if headers[name] ~= nil or name == "host" then
```

Comment:
Just noticed this: I think this `lower(name)` needs to stay, right? (it is taking the input from the user's conf, so that may include uppercase). The headers coming from ngx are the ones that are always normalized to lowercase.

Reply:
You're right, thanks for pointing that out.